app/config/config_test.yml

```
@@ -1,6 +1,7 @@
 parameters:
     AdapterSecurityAdminClass:                LegacyTests\PrestaShopBundle\Mock\AdapterSecurityAdminMock
     prestashop.security.voter.product.class:  LegacyTests\PrestaShopBundle\Mock\PageVoter
+    database_name: "test_prestashop"
```

Why hard code this?

---

Was already half hard coded here https://github.com/PrestaShop/PrestaShop/pull/15902/files#diff-e24c8c5370061c36a5415c30ab98c8b4L56 :sweat_smile: 

And we can't add a circular reference 
```suggestion
    database_name: "test_%database_name%"
```

We should do a new PR to make improve these configuration variables.

---

But we still must doing it for https://github.com/PrestaShop/PrestaShop/pull/15902/files#diff-2bad73d472db80e76e124fbfe9908964R100 because of legacy context and we must not load Symfony in front context right now :sob:

---

I changed it for https://github.com/PrestaShop/PrestaShop/pull/15902/commits/8b8585743b5ee146cc1811c0fa0e283732d64501 because it's currently not possible to have different parameters depending on the env.